### DISCUSSION TITLE
明年规划做一个低代码生成器

### DISCUSSION THREAD

**Discussion by chengjunjian2020** [shengxinjing/it-roadmap#36] [提需求]
我在公司大部分是做可视化大屏面对重复性代码和改需求是很头痛的，
我在想做一个可视化echarts拖拽的编辑器并且能够生成vue项目，后期支持扩展自定义组件拓展比如表格，这些不知道市面上公司内部有没有开发这种，我想尝试一下，觉得能够减少大多数工作量

**Comment by aehyok** [comment]
@chengjunjian2020 很不错的想法，目前我也有这方面的考虑，而且市面上也有很多这样的产品工具。

**Reply by chengjunjian2020** [reply]
要不加个联系方式，私底下沟通下

**Reply by aehyok** [reply]
我的微信aehyok

**Reply by aehyok** [reply]
@chengjunjian2020
